style.css

```diff
@@ -0,0 +1,104 @@
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+a { text-decoration: none; }
+
+.text {
+  width: 327px;
+  height: 150px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.icons {
+  padding: 2px;
+  margin: 0;
+  justify-content: space-between;
+  display: flex;
+  width: 50%;
+}
+
+.icon {
+  justify-content: space-between;
+}
+
+.head-section {
+  width: 327px;
+  height: 146px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.lets {
+  width: 117px;
+  height: 24px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.background-image {
+  position: absolute;
+  width: 462px;
+  height: 587.15px;
+  left: -34px;
+  top: 104px;
+}
```

Comment:
- [ Optional]  -  [ ] You've really done an awesome job i must admit 👏 , but these requirements are put in place with the aim of training your design skills to meet clients' demands as a developer. check the two screenshots I compare below, and you'll agree with me they aren't the same, just as the previous reviewers pointed out. kindly check the exact dimension given on the figma design, and let's wrap this up considering the good job you've done so far.
  ## Figma design
  ![image](https://user-images.githubusercontent.com/50990925/211839434-de47ff1a-f767-47bc-b553-4c99446879c7.png)

## Your Design
![image](https://user-images.githubusercontent.com/50990925/211839874-c689a0b1-01ee-47c2-bd36-23d3867e4aab.png)